docs/docs/guides/project-plugins.md

```diff
@@ -95,3 +95,31 @@ Depending on how you are building your package, you can register your project pl
         }
     )
     ```
+
+### Configuration for Project Plugins
+
+The `!config` YAML tag followed by a key can be used to provide end-user configurable values for project plugin pipelines.
+
+#### Plugin Creator Provided
+=== "plugin-pipeline.yaml"
+    ```
+    - implementation: myPlugin.testPipeline:TestExctractor
+      arguments:
+        base_url: !config 'base_url'
```

Comment:
Should we change the name of the var to not match the argument so it is explicit about them being different between argument and config value?
```suggestion
        base_url: !config 'config_base_url'
```

then in plugin_config, reference "config_base_url"

Reply:
yeah, i'll update